addons/portal/static/src/js/portal_chatter.js

```diff
@@ -254,6 +257,25 @@ var PortalChatter = publicWidget.Widget.extend({
         var page = $(ev.currentTarget).data('page');
         this._changeCurrentPage(page);
     },
+
+    /**
+     * @private
+     */
+    _onClickToggleInternal: function (ev) {
+        ev.preventDefault();
+
+        var $elem = $(ev.currentTarget);
+        this._rpc({
+            route: '/mail/toggle_internal',
+            params: {
+                message_id: $elem.data('message-id'),
+            },
+        })
+        .then(function (result) {
```

Comment:
I think then is usually not written on a new line compared to the `)`.

Reply:
Done